### ISSUE TITLE
Re implement printf

### ISSUE THREAD

**Issue by ParadoxZero** [ParadoxZero/paradoxOS#4]
The current implementation of `printf` leaves much to be desired.
Re-implement it to a better version.

**Comment by Lakshit-Chiranjiv** [comment]
So , you mean a modified version of printf using an user defined function???

**Comment by ParadoxZero** [comment]
Nope, talking about the libk version of `printf` kernel functions to be used before user mode exists.
